core/templates/base-components/base-content.component.css

```diff
@@ -0,0 +1,82 @@
+/*
+  NOTE to developers: Only the '.css' files should be edited. The styling should
```

Comment:
Arguably the .rtl.css file is also a ".css" file. Maybe say: "Only the files ending with '.css' (but not '.rtl.css') should be edited."

Ditto elsewhere.

Reply:
Done